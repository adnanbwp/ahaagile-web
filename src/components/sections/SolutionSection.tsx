import React from 'react';
import { Card, CardContent } from '@/components/ui/card';
import { Button } from '@/components/ui/button';
import { Zap, BarChart3, Shield, TrendingUp, Star } from 'lucide-react';

interface SolutionSectionProps {
  className?: string;
}

interface MetricProps {
  value: string;
  label: string;
  description: string;
}

interface SolutionCardProps {
  icon: React.ComponentType<{ className?: string }>;
  title: string;
  description: string;
  benefits: string[];
  index: number;
}

// Sub-component for individual metric display
function MetricCard({ value, label, description }: MetricProps): JSX.Element {
  return (
    <div 
      className="text-center p-8 rounded-2xl"
      style={{ background: 'var(--gradient-navy-light)' }}
    >
      <div className="text-5xl md:text-6xl font-inter font-bold text-white mb-2">
        {value}
      </div>
      <div className="text-xl font-inter font-semibold text-brand-yellow mb-1">
        {label}
      </div>
      <div className="text-gray-200 text-sm">
        {description}
      </div>
    </div>
  );
}

// Sub-component for individual solution card
function SolutionCard({ icon: IconComponent, title, description, benefits, index }: SolutionCardProps): JSX.Element {
  return (
    <Card 
      key={index}
      className="card-elevated hover:shadow-xl transition-all duration-300 hover:scale-105 group"
    >
      <CardContent className="p-6 text-center h-full flex flex-col">
        <div className="w-16 h-16 mx-auto mb-4 bg-gradient-accent rounded-full flex items-center justify-center group-hover:scale-110 transition-transform duration-300">
          <IconComponent className="w-8 h-8 text-brand-navy" />
        </div>
        <h3 className="text-xl font-inter font-semibold text-brand-navy mb-3">
          {title}
        </h3>
        <p className="text-secondary-600 leading-relaxed mb-4 flex-grow">
          {description}
        </p>
        <ul className="text-sm text-secondary-700 space-y-1">
          {benefits.map((benefit, benefitIndex) => (
            <li key={benefitIndex} className="flex items-center justify-center">
              <span className="w-1.5 h-1.5 bg-brand-yellow rounded-full mr-2"></span>
              {benefit}
            </li>
          ))}
        </ul>
      </CardContent>
    </Card>
  );
}

export default function SolutionSection({ className = '' }: SolutionSectionProps): JSX.Element {
  const solutions = [
    {
      icon: Zap,
      title: 'Process Automation',
      description: 'Streamline repetitive tasks and workflows to eliminate manual bottlenecks and increase efficiency',
      benefits: [
        'Reduce manual work by 80%',
        'Eliminate process bottlenecks',
        'Increase team productivity'
      ]
    },
    {
      icon: BarChart3,
      title: 'Smart Analytics',
      description: 'Transform data into actionable insights with intelligent reporting and performance tracking',
      benefits: [
        'Real-time performance dashboards',
        'Predictive insights & trends',
        'Data-driven decision making'
      ]
    },
    {
      icon: Shield,
      title: 'Compliance Management',
      description: 'Ensure regulatory adherence with automated compliance monitoring and reporting systems',
      benefits: [
        'Automated compliance tracking',
        'Risk mitigation protocols',
        'Audit-ready documentation'
      ]
    },
    {
      icon: TrendingUp,
      title: 'ROI Optimization',
      description: 'Maximize return on investment through strategic workflow improvements and cost reduction initiatives',
      benefits: [
        'Cost reduction strategies',
        'Performance optimization',
        'Measurable ROI improvements'
      ]
    }
  ];

  const metrics = [
    {
      value: '150+',
      label: 'Hours',
      description: 'Saved per month through automation'
    },
    {
      value: '$330K',
      label: 'Cost Savings',
      description: 'Average annual efficiency gains'
    },
    {
      value: '85%',
      label: 'Efficiency',
      description: 'Improvement in workflow performance'
    }
  ];

  return (
    <section className={`section-spacing bg-white ${className}`}>
      <div className="container-brand">
        {/* Solution Cards Section */}
        <div className="text-center mb-16">
          <h2 className="text-hero text-brand-navy mb-8">
            Transform Your Workflow Efficiency
          </h2>
          <p className="text-subhero max-w-3xl mx-auto">
            Our intelligent automation solutions address your specific challenges with proven methodologies 
            that deliver measurable results and sustainable growth.
          </p>
        </div>

        {/* Solution Cards Grid */}
        <div className="grid grid-cols-1 md:grid-cols-2 lg:grid-cols-4 gap-6 mb-20">
          {solutions.map((solution, index) => (
            <SolutionCard
              key={index}
              icon={solution.icon}
              title={solution.title}
              description={solution.description}
              benefits={solution.benefits}
              index={index}
            />
          ))}
        </div>

        {/* Social Proof Section */}
        <div className="text-center mb-16">
          <h2 className="text-4xl md:text-5xl font-inter font-bold text-brand-navy mb-12">
            Real Impact, Measurable Results
          </h2>
          
          {/* Metrics Display */}
          <div className="grid grid-cols-1 md:grid-cols-3 gap-8 mb-16">
            {metrics.map((metric, index) => (
              <MetricCard
                key={index}
                value={metric.value}
                label={metric.label}
                description={metric.description}
              />
            ))}
          </div>

          {/* Customer Testimonial */}
          <div className="max-w-4xl mx-auto bg-secondary-50 rounded-2xl p-8 md:p-12">
            <div className="flex justify-center mb-4">
              {[...Array(5)].map((_, i) => (
                <Star key={i} className="w-6 h-6 text-brand-yellow fill-current" />
              ))}
            </div>
            <blockquote className="text-2xl md:text-3xl font-serif text-brand-navy mb-6 leading-relaxed">
              &ldquo;The workflow automation transformed our operations completely. We&rsquo;ve eliminated countless hours of manual work and our team can now focus on high-value strategic initiatives that drive real business growth.&rdquo;
            </blockquote>
            <div className="text-lg font-inter font-semibold text-secondary-700">
              Sarah Johnson, Operations Director
            </div>
            <div className="text-secondary-600">
              Professional Services Firm
            </div>
          </div>
        </div>

        {/* Final CTA Section */}
        <div 
          className="text-center py-16 px-8 rounded-2xl"
          style={{ backgroundColor: 'var(--brand-yellow-light)' }}
        >
          <h2 className="text-4xl md:text-5xl font-inter font-bold text-brand-navy mb-6">
            Ready to Transform Your Workflows?
          </h2>
          <p className="text-xl text-brand-navy mb-8 max-w-2xl mx-auto leading-relaxed">
            Join hundreds of professional services firms who&rsquo;ve already eliminated inefficiencies 
            and unlocked their team&rsquo;s full potential.
          </p>
          <div className="flex flex-col sm:flex-row gap-4 justify-center items-center">
            <Button 
              className="btn-primary px-8 py-4 text-lg font-semibold focus-brand"
              aria-label="Book a consultation to discuss your workflow automation needs"
            >
              Book Your Strategy Session
            </Button>
            <Button 
              variant="outline"
              className="btn-secondary px-8 py-4 text-lg font-semibold focus-brand"
              aria-label="View case study to see real results from workflow automation"
            >
              View Case Study
            </Button>
          </div>
        </div>
      </div>
    </section>
  );
}