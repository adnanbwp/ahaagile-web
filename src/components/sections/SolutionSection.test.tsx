import { render, screen } from '@testing-library/react';
import '@testing-library/jest-dom';
import SolutionSection from './SolutionSection';

describe('SolutionSection', () => {
  beforeEach(() => {
    render(<SolutionSection />);
  });

  describe('Solution Cards Section', () => {
    it('renders the main section header', () => {
      expect(screen.getByRole('heading', { name: /transform your workflow efficiency/i })).toBeInTheDocument();
    });

    it('renders section description', () => {
      expect(screen.getByText(/our intelligent automation solutions address your specific challenges/i)).toBeInTheDocument();
    });

    it('renders all four solution cards with correct titles', () => {
      expect(screen.getByRole('heading', { name: /process automation/i })).toBeInTheDocument();
      expect(screen.getByRole('heading', { name: /smart analytics/i })).toBeInTheDocument();
      expect(screen.getByRole('heading', { name: /compliance management/i })).toBeInTheDocument();
      expect(screen.getByRole('heading', { name: /roi optimization/i })).toBeInTheDocument();
    });

    it('renders solution card descriptions', () => {
      expect(screen.getByText(/streamline repetitive tasks and workflows/i)).toBeInTheDocument();
      expect(screen.getByText(/transform data into actionable insights/i)).toBeInTheDocument();
      expect(screen.getByText(/ensure regulatory adherence with automated compliance/i)).toBeInTheDocument();
      expect(screen.getByText(/maximize return on investment through strategic workflow/i)).toBeInTheDocument();
    });

    it('renders benefit points for each solution card', () => {
      expect(screen.getByText(/reduce manual work by 80%/i)).toBeInTheDocument();
      expect(screen.getByText(/real-time performance dashboards/i)).toBeInTheDocument();
      expect(screen.getByText(/automated compliance tracking/i)).toBeInTheDocument();
      expect(screen.getByText(/cost reduction strategies/i)).toBeInTheDocument();
    });

    it('applies hover animation classes to solution cards', () => {
      const cards = document.querySelectorAll('.card-elevated');
      expect(cards).toHaveLength(4);
      cards.forEach(card => {
        expect(card).toHaveClass('hover:shadow-xl', 'transition-all', 'duration-300', 'hover:scale-105', 'group');
      });
    });
  });

  describe('Social Proof Section', () => {
    it('renders social proof section header', () => {
      expect(screen.getByRole('heading', { name: /real impact, measurable results/i })).toBeInTheDocument();
    });

    it('renders all three key metrics', () => {
      expect(screen.getByText('150+')).toBeInTheDocument();
      expect(screen.getByText('$330K')).toBeInTheDocument();
      expect(screen.getByText('85%')).toBeInTheDocument();
    });

    it('renders metric labels and descriptions', () => {
      expect(screen.getByText('Hours')).toBeInTheDocument();
      expect(screen.getByText('Cost Savings')).toBeInTheDocument();
      expect(screen.getByText('Efficiency')).toBeInTheDocument();
      expect(screen.getByText(/saved per month through automation/i)).toBeInTheDocument();
      expect(screen.getByText(/average annual efficiency gains/i)).toBeInTheDocument();
      expect(screen.getByText(/improvement in workflow performance/i)).toBeInTheDocument();
    });

    it('applies gradient background styling to metrics', () => {
      const metricCards = screen.getAllByText('150+').map(el => el.closest('div'));
      metricCards.forEach(card => {
        if (card && card.style.background) {
          expect(card.style.background).toContain('linear-gradient');
        }
      });
    });
  });

  describe('Customer Testimonial', () => {
    it('renders star rating with 5 stars', () => {
      // Stars are rendered as SVG icons, so we check for their presence
      const stars = document.querySelectorAll('[data-testid="star"]') || 
                   document.querySelectorAll('svg[data-lucide="star"]');
      
      // We expect at least the star elements to be present (5 stars)
      const allSvgs = document.querySelectorAll('svg');
      expect(allSvgs.length).toBeGreaterThanOrEqual(5);
      expect(document.querySelector('svg')).toBeInTheDocument();
    });

    it('renders customer testimonial quote', () => {
      expect(screen.getByText(/the workflow automation transformed our operations completely/i)).toBeInTheDocument();
    });

    it('renders customer attribution', () => {
      expect(screen.getByText(/sarah johnson/i)).toBeInTheDocument();
      expect(screen.getByText(/operations director/i)).toBeInTheDocument();
      
      // Check specifically within the testimonial section to avoid duplicate matches
      const testimonialSection = document.querySelector('.max-w-4xl.mx-auto.bg-secondary-50');
      expect(testimonialSection).toBeInTheDocument();
      expect(testimonialSection).toHaveTextContent('Professional Services Firm');
    });

    it('applies proper semantic structure for testimonial', () => {
      const blockquote = screen.getByRole('blockquote') || document.querySelector('blockquote');
      expect(blockquote).toBeInTheDocument();
    });
  });

  describe('Final CTA Section', () => {
    it('renders CTA section header', () => {
      expect(screen.getByRole('heading', { name: /ready to transform your workflows/i })).toBeInTheDocument();
    });

    it('renders CTA description', () => {
      expect(screen.getByText(/join hundreds of professional services firms/i)).toBeInTheDocument();
    });

    it('renders primary CTA button', () => {
      expect(screen.getByText(/book your strategy session/i)).toBeInTheDocument();
    });

    it('renders secondary CTA button', () => {
      expect(screen.getByText(/view case study/i)).toBeInTheDocument();
    });

    it('applies yellow accent background to CTA section', () => {
      const ctaSection = screen.getByRole('heading', { name: /ready to transform your workflows/i }).closest('div');
      expect(ctaSection).toHaveStyle({ backgroundColor: 'var(--brand-yellow-light)' });
    });

    it('includes proper accessibility features for CTA buttons', () => {
      const buttons = screen.getAllByRole('button');
      expect(buttons).toHaveLength(2);
      
      buttons.forEach(button => {
        expect(button).toHaveAttribute('aria-label');
      });
    });
  });

  describe('Responsive Design', () => {
    it('applies responsive grid classes to solution cards', () => {
      const grid = document.querySelector('.grid');
      expect(grid).toHaveClass('grid-cols-1', 'md:grid-cols-2', 'lg:grid-cols-4');
    });

    it('applies responsive grid classes to metrics', () => {
      const metricsGrid = screen.getByText('150+').closest('.grid');
      expect(metricsGrid).toHaveClass('grid-cols-1', 'md:grid-cols-3');
    });

    it('applies responsive text sizing to headers', () => {
      const heroText = screen.getByRole('heading', { name: /transform your workflow efficiency/i });
      expect(heroText).toHaveClass('text-hero');
      
      const ctaText = screen.getByRole('heading', { name: /ready to transform your workflows/i });
      expect(ctaText).toHaveClass('text-4xl', 'md:text-5xl');
    });
  });

  describe('Accessibility', () => {
    it('has proper semantic HTML structure', () => {
      const section = document.querySelector('section');
      expect(section).toBeInTheDocument();
    });

    it('includes proper heading hierarchy', () => {
      const headings = screen.getAllByRole('heading');
      expect(headings.length).toBeGreaterThan(0);
      
      // Check that we have h2 and h3 elements for proper hierarchy
      const h2Elements = headings.filter(h => h.tagName === 'H2');
      const h3Elements = headings.filter(h => h.tagName === 'H3');
      expect(h2Elements.length).toBeGreaterThan(0);
      expect(h3Elements.length).toBeGreaterThan(0);
    });

    it('includes focus management classes for buttons', () => {
      const buttons = screen.getAllByRole('button');
      buttons.forEach(button => {
        expect(button).toHaveClass('focus-brand');
      });
    });
  });

  describe('Component Props', () => {
    it('applies custom className when provided', () => {
      const { container } = render(<SolutionSection className="custom-class" />);
      expect(container.firstChild).toHaveClass('custom-class');
    });

    it('applies default classes when no className provided', () => {
      const { container } = render(<SolutionSection />);
      expect(container.firstChild).toHaveClass('section-spacing', 'bg-white');
    });
  });
});

describe('SolutionSection Snapshot', () => {
  it('matches snapshot to prevent visual regression', () => {
    const { container } = render(<SolutionSection />);
    expect(container.firstChild).toMatchSnapshot();
  });
});