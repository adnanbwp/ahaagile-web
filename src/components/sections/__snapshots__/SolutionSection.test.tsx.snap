// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SolutionSection Snapshot matches snapshot to prevent visual regression 1`] = `
<section
  class="section-spacing bg-white "
>
  <div
    class="container-brand"
  >
    <div
      class="text-center mb-16"
    >
      <h2
        class="text-hero text-brand-navy mb-8"
      >
        Transform Your Workflow Efficiency
      </h2>
      <p
        class="text-subhero max-w-3xl mx-auto"
      >
        Our intelligent automation solutions address your specific challenges with proven methodologies that deliver measurable results and sustainable growth.
      </p>
    </div>
    <div
      class="grid grid-cols-1 md:grid-cols-2 lg:grid-cols-4 gap-6 mb-20"
    >
      <div
        class="rounded-lg border bg-card text-card-foreground shadow-sm card-elevated hover:shadow-xl transition-all duration-300 hover:scale-105 group"
      >
        <div
          class="p-6 text-center h-full flex flex-col"
        >
          <div
            class="w-16 h-16 mx-auto mb-4 bg-gradient-accent rounded-full flex items-center justify-center group-hover:scale-110 transition-transform duration-300"
          >
            <svg
              aria-hidden="true"
              class="lucide lucide-zap w-8 h-8 text-brand-navy"
              fill="none"
              height="24"
              stroke="currentColor"
              stroke-linecap="round"
              stroke-linejoin="round"
              stroke-width="2"
              viewBox="0 0 24 24"
              width="24"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M4 14a1 1 0 0 1-.78-1.63l9.9-10.2a.5.5 0 0 1 .86.46l-1.92 6.02A1 1 0 0 0 13 10h7a1 1 0 0 1 .78 1.63l-9.9 10.2a.5.5 0 0 1-.86-.46l1.92-6.02A1 1 0 0 0 11 14z"
              />
            </svg>
          </div>
          <h3
            class="text-xl font-inter font-semibold text-brand-navy mb-3"
          >
            Process Automation
          </h3>
          <p
            class="text-secondary-600 leading-relaxed mb-4 flex-grow"
          >
            Streamline repetitive tasks and workflows to eliminate manual bottlenecks and increase efficiency
          </p>
          <ul
            class="text-sm text-secondary-700 space-y-1"
          >
            <li
              class="flex items-center justify-center"
            >
              <span
                class="w-1.5 h-1.5 bg-brand-yellow rounded-full mr-2"
              />
              Reduce manual work by 80%
            </li>
            <li
              class="flex items-center justify-center"
            >
              <span
                class="w-1.5 h-1.5 bg-brand-yellow rounded-full mr-2"
              />
              Eliminate process bottlenecks
            </li>
            <li
              class="flex items-center justify-center"
            >
              <span
                class="w-1.5 h-1.5 bg-brand-yellow rounded-full mr-2"
              />
              Increase team productivity
            </li>
          </ul>
        </div>
      </div>
      <div
        class="rounded-lg border bg-card text-card-foreground shadow-sm card-elevated hover:shadow-xl transition-all duration-300 hover:scale-105 group"
      >
        <div
          class="p-6 text-center h-full flex flex-col"
        >
          <div
            class="w-16 h-16 mx-auto mb-4 bg-gradient-accent rounded-full flex items-center justify-center group-hover:scale-110 transition-transform duration-300"
          >
            <svg
              aria-hidden="true"
              class="lucide lucide-chart-column w-8 h-8 text-brand-navy"
              fill="none"
              height="24"
              stroke="currentColor"
              stroke-linecap="round"
              stroke-linejoin="round"
              stroke-width="2"
              viewBox="0 0 24 24"
              width="24"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M3 3v16a2 2 0 0 0 2 2h16"
              />
              <path
                d="M18 17V9"
              />
              <path
                d="M13 17V5"
              />
              <path
                d="M8 17v-3"
              />
            </svg>
          </div>
          <h3
            class="text-xl font-inter font-semibold text-brand-navy mb-3"
          >
            Smart Analytics
          </h3>
          <p
            class="text-secondary-600 leading-relaxed mb-4 flex-grow"
          >
            Transform data into actionable insights with intelligent reporting and performance tracking
          </p>
          <ul
            class="text-sm text-secondary-700 space-y-1"
          >
            <li
              class="flex items-center justify-center"
            >
              <span
                class="w-1.5 h-1.5 bg-brand-yellow rounded-full mr-2"
              />
              Real-time performance dashboards
            </li>
            <li
              class="flex items-center justify-center"
            >
              <span
                class="w-1.5 h-1.5 bg-brand-yellow rounded-full mr-2"
              />
              Predictive insights & trends
            </li>
            <li
              class="flex items-center justify-center"
            >
              <span
                class="w-1.5 h-1.5 bg-brand-yellow rounded-full mr-2"
              />
              Data-driven decision making
            </li>
          </ul>
        </div>
      </div>
      <div
        class="rounded-lg border bg-card text-card-foreground shadow-sm card-elevated hover:shadow-xl transition-all duration-300 hover:scale-105 group"
      >
        <div
          class="p-6 text-center h-full flex flex-col"
        >
          <div
            class="w-16 h-16 mx-auto mb-4 bg-gradient-accent rounded-full flex items-center justify-center group-hover:scale-110 transition-transform duration-300"
          >
            <svg
              aria-hidden="true"
              class="lucide lucide-shield w-8 h-8 text-brand-navy"
              fill="none"
              height="24"
              stroke="currentColor"
              stroke-linecap="round"
              stroke-linejoin="round"
              stroke-width="2"
              viewBox="0 0 24 24"
              width="24"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M20 13c0 5-3.5 7.5-7.66 8.95a1 1 0 0 1-.67-.01C7.5 20.5 4 18 4 13V6a1 1 0 0 1 1-1c2 0 4.5-1.2 6.24-2.72a1.17 1.17 0 0 1 1.52 0C14.51 3.81 17 5 19 5a1 1 0 0 1 1 1z"
              />
            </svg>
          </div>
          <h3
            class="text-xl font-inter font-semibold text-brand-navy mb-3"
          >
            Compliance Management
          </h3>
          <p
            class="text-secondary-600 leading-relaxed mb-4 flex-grow"
          >
            Ensure regulatory adherence with automated compliance monitoring and reporting systems
          </p>
          <ul
            class="text-sm text-secondary-700 space-y-1"
          >
            <li
              class="flex items-center justify-center"
            >
              <span
                class="w-1.5 h-1.5 bg-brand-yellow rounded-full mr-2"
              />
              Automated compliance tracking
            </li>
            <li
              class="flex items-center justify-center"
            >
              <span
                class="w-1.5 h-1.5 bg-brand-yellow rounded-full mr-2"
              />
              Risk mitigation protocols
            </li>
            <li
              class="flex items-center justify-center"
            >
              <span
                class="w-1.5 h-1.5 bg-brand-yellow rounded-full mr-2"
              />
              Audit-ready documentation
            </li>
          </ul>
        </div>
      </div>
      <div
        class="rounded-lg border bg-card text-card-foreground shadow-sm card-elevated hover:shadow-xl transition-all duration-300 hover:scale-105 group"
      >
        <div
          class="p-6 text-center h-full flex flex-col"
        >
          <div
            class="w-16 h-16 mx-auto mb-4 bg-gradient-accent rounded-full flex items-center justify-center group-hover:scale-110 transition-transform duration-300"
          >
            <svg
              aria-hidden="true"
              class="lucide lucide-trending-up w-8 h-8 text-brand-navy"
              fill="none"
              height="24"
              stroke="currentColor"
              stroke-linecap="round"
              stroke-linejoin="round"
              stroke-width="2"
              viewBox="0 0 24 24"
              width="24"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M16 7h6v6"
              />
              <path
                d="m22 7-8.5 8.5-5-5L2 17"
              />
            </svg>
          </div>
          <h3
            class="text-xl font-inter font-semibold text-brand-navy mb-3"
          >
            ROI Optimization
          </h3>
          <p
            class="text-secondary-600 leading-relaxed mb-4 flex-grow"
          >
            Maximize return on investment through strategic workflow improvements and cost reduction initiatives
          </p>
          <ul
            class="text-sm text-secondary-700 space-y-1"
          >
            <li
              class="flex items-center justify-center"
            >
              <span
                class="w-1.5 h-1.5 bg-brand-yellow rounded-full mr-2"
              />
              Cost reduction strategies
            </li>
            <li
              class="flex items-center justify-center"
            >
              <span
                class="w-1.5 h-1.5 bg-brand-yellow rounded-full mr-2"
              />
              Performance optimization
            </li>
            <li
              class="flex items-center justify-center"
            >
              <span
                class="w-1.5 h-1.5 bg-brand-yellow rounded-full mr-2"
              />
              Measurable ROI improvements
            </li>
          </ul>
        </div>
      </div>
    </div>
    <div
      class="text-center mb-16"
    >
      <h2
        class="text-4xl md:text-5xl font-inter font-bold text-brand-navy mb-12"
      >
        Real Impact, Measurable Results
      </h2>
      <div
        class="grid grid-cols-1 md:grid-cols-3 gap-8 mb-16"
      >
        <div
          class="text-center p-8 rounded-2xl"
          style="background: var(--gradient-navy-light);"
        >
          <div
            class="text-5xl md:text-6xl font-inter font-bold text-white mb-2"
          >
            150+
          </div>
          <div
            class="text-xl font-inter font-semibold text-brand-yellow mb-1"
          >
            Hours
          </div>
          <div
            class="text-gray-200 text-sm"
          >
            Saved per month through automation
          </div>
        </div>
        <div
          class="text-center p-8 rounded-2xl"
          style="background: var(--gradient-navy-light);"
        >
          <div
            class="text-5xl md:text-6xl font-inter font-bold text-white mb-2"
          >
            $330K
          </div>
          <div
            class="text-xl font-inter font-semibold text-brand-yellow mb-1"
          >
            Cost Savings
          </div>
          <div
            class="text-gray-200 text-sm"
          >
            Average annual efficiency gains
          </div>
        </div>
        <div
          class="text-center p-8 rounded-2xl"
          style="background: var(--gradient-navy-light);"
        >
          <div
            class="text-5xl md:text-6xl font-inter font-bold text-white mb-2"
          >
            85%
          </div>
          <div
            class="text-xl font-inter font-semibold text-brand-yellow mb-1"
          >
            Efficiency
          </div>
          <div
            class="text-gray-200 text-sm"
          >
            Improvement in workflow performance
          </div>
        </div>
      </div>
      <div
        class="max-w-4xl mx-auto bg-secondary-50 rounded-2xl p-8 md:p-12"
      >
        <div
          class="flex justify-center mb-4"
        >
          <svg
            aria-hidden="true"
            class="lucide lucide-star w-6 h-6 text-brand-yellow fill-current"
            fill="none"
            height="24"
            stroke="currentColor"
            stroke-linecap="round"
            stroke-linejoin="round"
            stroke-width="2"
            viewBox="0 0 24 24"
            width="24"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M11.525 2.295a.53.53 0 0 1 .95 0l2.31 4.679a2.123 2.123 0 0 0 1.595 1.16l5.166.756a.53.53 0 0 1 .294.904l-3.736 3.638a2.123 2.123 0 0 0-.611 1.878l.882 5.14a.53.53 0 0 1-.771.56l-4.618-2.428a2.122 2.122 0 0 0-1.973 0L6.396 21.01a.53.53 0 0 1-.77-.56l.881-5.139a2.122 2.122 0 0 0-.611-1.879L2.16 9.795a.53.53 0 0 1 .294-.906l5.165-.755a2.122 2.122 0 0 0 1.597-1.16z"
            />
          </svg>
          <svg
            aria-hidden="true"
            class="lucide lucide-star w-6 h-6 text-brand-yellow fill-current"
            fill="none"
            height="24"
            stroke="currentColor"
            stroke-linecap="round"
            stroke-linejoin="round"
            stroke-width="2"
            viewBox="0 0 24 24"
            width="24"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M11.525 2.295a.53.53 0 0 1 .95 0l2.31 4.679a2.123 2.123 0 0 0 1.595 1.16l5.166.756a.53.53 0 0 1 .294.904l-3.736 3.638a2.123 2.123 0 0 0-.611 1.878l.882 5.14a.53.53 0 0 1-.771.56l-4.618-2.428a2.122 2.122 0 0 0-1.973 0L6.396 21.01a.53.53 0 0 1-.77-.56l.881-5.139a2.122 2.122 0 0 0-.611-1.879L2.16 9.795a.53.53 0 0 1 .294-.906l5.165-.755a2.122 2.122 0 0 0 1.597-1.16z"
            />
          </svg>
          <svg
            aria-hidden="true"
            class="lucide lucide-star w-6 h-6 text-brand-yellow fill-current"
            fill="none"
            height="24"
            stroke="currentColor"
            stroke-linecap="round"
            stroke-linejoin="round"
            stroke-width="2"
            viewBox="0 0 24 24"
            width="24"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M11.525 2.295a.53.53 0 0 1 .95 0l2.31 4.679a2.123 2.123 0 0 0 1.595 1.16l5.166.756a.53.53 0 0 1 .294.904l-3.736 3.638a2.123 2.123 0 0 0-.611 1.878l.882 5.14a.53.53 0 0 1-.771.56l-4.618-2.428a2.122 2.122 0 0 0-1.973 0L6.396 21.01a.53.53 0 0 1-.77-.56l.881-5.139a2.122 2.122 0 0 0-.611-1.879L2.16 9.795a.53.53 0 0 1 .294-.906l5.165-.755a2.122 2.122 0 0 0 1.597-1.16z"
            />
          </svg>
          <svg
            aria-hidden="true"
            class="lucide lucide-star w-6 h-6 text-brand-yellow fill-current"
            fill="none"
            height="24"
            stroke="currentColor"
            stroke-linecap="round"
            stroke-linejoin="round"
            stroke-width="2"
            viewBox="0 0 24 24"
            width="24"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M11.525 2.295a.53.53 0 0 1 .95 0l2.31 4.679a2.123 2.123 0 0 0 1.595 1.16l5.166.756a.53.53 0 0 1 .294.904l-3.736 3.638a2.123 2.123 0 0 0-.611 1.878l.882 5.14a.53.53 0 0 1-.771.56l-4.618-2.428a2.122 2.122 0 0 0-1.973 0L6.396 21.01a.53.53 0 0 1-.77-.56l.881-5.139a2.122 2.122 0 0 0-.611-1.879L2.16 9.795a.53.53 0 0 1 .294-.906l5.165-.755a2.122 2.122 0 0 0 1.597-1.16z"
            />
          </svg>
          <svg
            aria-hidden="true"
            class="lucide lucide-star w-6 h-6 text-brand-yellow fill-current"
            fill="none"
            height="24"
            stroke="currentColor"
            stroke-linecap="round"
            stroke-linejoin="round"
            stroke-width="2"
            viewBox="0 0 24 24"
            width="24"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M11.525 2.295a.53.53 0 0 1 .95 0l2.31 4.679a2.123 2.123 0 0 0 1.595 1.16l5.166.756a.53.53 0 0 1 .294.904l-3.736 3.638a2.123 2.123 0 0 0-.611 1.878l.882 5.14a.53.53 0 0 1-.771.56l-4.618-2.428a2.122 2.122 0 0 0-1.973 0L6.396 21.01a.53.53 0 0 1-.77-.56l.881-5.139a2.122 2.122 0 0 0-.611-1.879L2.16 9.795a.53.53 0 0 1 .294-.906l5.165-.755a2.122 2.122 0 0 0 1.597-1.16z"
            />
          </svg>
        </div>
        <blockquote
          class="text-2xl md:text-3xl font-serif text-brand-navy mb-6 leading-relaxed"
        >
          “The workflow automation transformed our operations completely. We’ve eliminated countless hours of manual work and our team can now focus on high-value strategic initiatives that drive real business growth.”
        </blockquote>
        <div
          class="text-lg font-inter font-semibold text-secondary-700"
        >
          Sarah Johnson, Operations Director
        </div>
        <div
          class="text-secondary-600"
        >
          Professional Services Firm
        </div>
      </div>
    </div>
    <div
      class="text-center py-16 px-8 rounded-2xl"
      style="background-color: var(--brand-yellow-light);"
    >
      <h2
        class="text-4xl md:text-5xl font-inter font-bold text-brand-navy mb-6"
      >
        Ready to Transform Your Workflows?
      </h2>
      <p
        class="text-xl text-brand-navy mb-8 max-w-2xl mx-auto leading-relaxed"
      >
        Join hundreds of professional services firms who’ve already eliminated inefficiencies and unlocked their team’s full potential.
      </p>
      <div
        class="flex flex-col sm:flex-row gap-4 justify-center items-center"
      >
        <button
          aria-label="Book a consultation to discuss your workflow automation needs"
          class="inline-flex items-center justify-center whitespace-nowrap rounded-md ring-offset-background transition-colors focus-visible:outline-none focus-visible:ring-2 focus-visible:ring-ring focus-visible:ring-offset-2 disabled:pointer-events-none disabled:opacity-50 bg-shadcn-primary text-shadcn-primary-foreground hover:bg-shadcn-primary/90 h-10 btn-primary px-8 py-4 text-lg font-semibold focus-brand"
        >
          Book Your Strategy Session
        </button>
        <button
          aria-label="View case study to see real results from workflow automation"
          class="inline-flex items-center justify-center whitespace-nowrap rounded-md ring-offset-background transition-colors focus-visible:outline-none focus-visible:ring-2 focus-visible:ring-ring focus-visible:ring-offset-2 disabled:pointer-events-none disabled:opacity-50 border border-input bg-background hover:bg-shadcn-accent hover:text-shadcn-accent-foreground h-10 btn-secondary px-8 py-4 text-lg font-semibold focus-brand"
        >
          View Case Study
        </button>
      </div>
    </div>
  </div>
</section>
`;
